vault_program/src/initialize_vault.rs

```diff
@@ -90,6 +104,25 @@ pub fn process_initialize_vault(
         )?;
     }
 
+    // Deposit min ST
+    {
+        invoke(
+            &transfer(
+                token_program.key,
+                admin_st_token_account.key,
+                vault_st_token_account.key,
+                admin.key,
+                &[],
+                Vault::INITIALIZATION_TOKEN_AMOUNT,
+            )?,
+            &[
+                vault_st_token_account.clone(),
+                admin_st_token_account.clone(),
```

Comment:
are these out of order? shouldn't it be source, dest

Reply:
either im misunderstanding or im surprised this is working lmao

Reply:
I honestly never thought the order mattered, I can swap them around and see if it breaks anything

Reply:
Order does not matter - but I switched it for ocd sake